elements/cloud-init-growpart-lvm/post-install.d/55-growpart-lvm

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+if [ ${DIB_DEBUG_TRACE:-1} -gt 0 ]; then
+    set -x
+fi
+set -eu
+set -o pipefail
+
+if [ -d /etc/cloud/cloud.cfg.d ]; then
+    cat > /etc/cloud/cloud.cfg.d/55-growpart-lvm.cfg <<EOF
+#cloud-config
+growpart:
+  mode: auto
+  devices:
+    - "/dev/vda3"
+    - "/dev/sda3"
```

Comment:
Does this mean that we need to know the name of the disks in advance? Does that mean that the image is tied to specific hardware?

Reply:
this was specifically tailored to our sms-lab disk-layout

Reply:
I have added a variable for partition list